src/main/java/com/myhome/controllers/response/AddCommunityAdminResponse.java

```diff
@@ -27,6 +28,7 @@
 @AllArgsConstructor
 @Getter
 @Setter
+@Data
```

Comment:
`@Data` includes `@Getter`, `@Setter`, `@RequiredArgsConstructor`, `@ToString`, `@EqualsAndHashCode` so you can remove  `@Getter` and `@Setter` annotations